src/main/resources/assets/skyblocker/lang/en_us.json

```diff
@@ -368,6 +368,54 @@
   "text.autoconfig.skyblocker.option.messages.hideDeath.@Tooltip": "Filters the player death messages from chat.",
   "text.autoconfig.skyblocker.option.messages.hideDicer": "Hide Dicer Messages",
   "text.autoconfig.skyblocker.option.messages.hideDicer.@Tooltip": "Filters the Dicer messages from chat.",
+  "text.autoconfig.skyblocker.option.messages.chatRules": "Custom Chat Rules",
+  "text.autoconfig.skyblocker.option.messages.chatRules.announcementLength": "Announcement Display Time",
+  "text.autoconfig.skyblocker.option.messages.chatRules.announcementLength.@Tooltip": "The amount of time the announcement will show for in ticks.",
+  "text.autoconfig.skyblocker.option.messages.chatRules.announcementScale": "Announcement Size",
+  "text.autoconfig.skyblocker.option.messages.chatRules.announcementScale.@Tooltip": "The amount to scale the size of the announcement.",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen": "Chat Rules Config...",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.new": "New Chat Rule",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleName": "Rule Name",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleEnabled": "Rule Enabled",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.modify": "Modify",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.editRule": "Edit",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.editRule.@Tooltip": "Open config for rule.",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleScreen": "Chat Rule Config...",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleScreen.true": "True",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleScreen.false": "False",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleScreen.inputs": "Inputs:",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleScreen.name": "Name:",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleScreen.name.@Tooltip": "The name of the rule.",
+  "text.autoconfig.skyblocker.option.messages.chatRules.screen.ruleScreen.filter": "ChatFilter:",
```

Comment:
add a space between the two words, maybe you could shorten it to just "Filter" too